### PR TITLE
Fix display names

### PR DESCRIPTION
**Pull request by lhbrennan**

### CODE REVIEW

File: src/badge/styled-components.ts [hunk 1]
```diff
@@ -184,8 +184,6 @@ export const StyledRoot = styled<'div', {}>('div', () => {
   };
 });
 
-StyledRoot.displayName = 'StyledRoot';
```
Comment (chasestarr) [inline src/badge/styled-components.ts:187]: Please keep the displayName assignments. Enzyme uses this property to search for components in the react tree

Comment (lhbrennan) [reply src/badge/styled-components.ts:187]: Ah. My understanding was that the display name is automatically deduced from the name of the function (or the variable the component is assigned to), and thus explicitly assigning displayName is only necessary when you want the component to show up differently in the dom. Is that incorrect?

Comment (chasestarr) [reply src/badge/styled-components.ts:187]: Not when building with styletron :(

Comment (lhbrennan) [reply src/badge/styled-components.ts:187]: Gotcha. Added displayName to all the recent new components.

**Review by chasestarr — APPROVED** [review]

**Pull request merged**